### PR TITLE
Use Invenia's PublicCI account for Travis, sending emails on failure

### PR DESCRIPTION
**Pull request by mattBrzezinski**

### CODE REVIEW

**Comment by mattBrzezinski** [comment]
bors try

File: .travis.yml [hunk 1]
```diff
@@ -7,17 +7,22 @@ julia:
...
 branches:
   only:
-    - master
-    - staging
-    - trying
-    - /^v[0-9]+\.[0-9]+\.[0-9]+$/  # version tags
+  - master
+  - staging
+  - trying
+  - "/^v[0-9]+\\.[0-9]+\\.[0-9]+$/"
```
Comment (mjram0s) [inline .travis.yml]: Maybe add back the comment?

**Review by mjram0s — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (mattBrzezinski) [reply .travis.yml]: Can do! This just happened when running:

> travis pro encrypt {creds} --add

It likes to re-format the file

**Comment by mattBrzezinski** [comment]
Bors is being weird, and is sitting here expecting a status but it ran successfully. Squashing and merging this in.

**Pull request merged**